docs/Chapter02-Overview-LoggingIn-and-Authentication.md

```diff
@@ -1,10 +1,8 @@
 # Security in the Newsroom: Who are you?
 
-This is a living document and the most recent version can be found at: <https://goo.gl/7ojKpw>. This lesson plan, which covers Passwords and Two Factor Authentication is a great hour long brown-bag session that was shared with us by Alan Palazzolo at the Minneapolis Star Tribune. For other great overview lesson plans, take a look at:
+This lesson plan, which covers Passwords and Two Factor Authentication is a great hour long brown-bag session that was shared with us by Alan Palazzolo at the Minneapolis Star Tribune. For other great overview lesson plans, take a look at:
```

Comment:
There's nothing linked or reference after "..take a look at:"

Reply:
Going to go ahead and drop that line, not sure what it refers to.